sktime/classification/kernel_based/_rocket_classifier.py

```diff
@@ -236,4 +247,4 @@ def get_test_params(cls, parameter_set="default"):
         if parameter_set == "results_comparison":
             return {"num_kernels": 100}
         else:
-            return {"num_kernels": 20}
+            return {"num_kernels": 20, "rocket_transform": "minirocket"}
```

Comment:
this should actually be a lits of multiple dictionaries, each being a parameter set that is tested.

ou can also omit the `else` for better readability, as it follows an `if`-`return` statement. If anything past the `if` is reached, we know it is `else` (logically)